include/deal.II/base/function_lib.h

```diff
@@ -1241,8 +1241,8 @@ namespace Functions
      * class has a number of conversion constructors that allow converting
      * other data types into a table where you specify this argument.
      */
-    InterpolatedUniformGridData (const std_cxx11::array<std::pair<double,double>,dim> &interval_endpoints,
-                                 const std_cxx11::array<unsigned int,dim>             &n_subintervals,
+    InterpolatedUniformGridData (const std::array<std::pair<double,double>,dim> &interval_endpoints,
+                                 const std::array<unsigned int,dim>             &n_subintervals,
                                  const Table<dim,double>                              &data_values);
```

Comment:
This issue wasn't addressed before the merge, but I didn't want to hold the issue up longer than necessary. I'll submit a follow-up patch.